components/common/Row.tsx

```diff
@@ -1,21 +1,17 @@
 import React from 'react';
-import type { StyleProp, ViewStyle } from 'react-native';
-import { StyleSheet, View as NativeView } from 'react-native';
+import { StyleSheet, View } from 'react-native';
 
 type RowProps = {
   children: React.ReactNode;
-  style?: StyleProp<ViewStyle>;
+  style?: object;
```

Comment:
Why did you loosen this type? `object` will allow any object, even invalid styles, to be passed and will not offer autocompletion of the CSS properties in the editor. We should keep it to `StyleProp<ViewStyle>`.

Reply:
fixed